resources/configurationFiles/analysisCopyNumberEstimation.xml

```diff
@@ -42,6 +43,7 @@
 
 
         <!-- Reference files -->
+        <cvalue name="sharedFilesBaseDirectory" value="/icgc/ngs_share" type="path"/>
```

Comment:
You probably use the base configuration in the ngs_share, right? Did you consider putting this into one of the files in there? It's not really appropriate in a plugin to have such location-specific information in default config.

Reply:
Good point. I removed this line but added the comment `<!-- sharedFilesBaseDirectory, inputBaseDirectory and outputBaseDirectory have to be set in the project.xml file -->` for clarity.
This is also mentioned in the documentation but I found it to be a little confusing so twice mentioning it should be fine.